index.js

```diff
@@ -1,26 +1,12 @@
-const merge = require('webpack-merge')
-const { applyConfig } = require('./lib/config-helpers')
 const { join } = require('path')
-
-const defaultSharedConfig = require('./webpack.shared.config')
-const defaultClientConfig = require('./webpack.client.config')
-const defaultServerConfig = require('./webpack.server.config')
-const defaultDevConfig = require('./webpack.dev.config')
+const configMerge = require('./lib/configMerge')
```

Comment:
@slwen and I did try this, but ran into spaghetti-type situations because the `serve` script also calls the `build` script. This solution is somewhat jankey but much simpler and easier to reason about.

Perhaps this could be addressed in a seperate PR. We are concerned with configuring the `lint` and `test` scripts. We are not refactoring the entire `build` and `serve` script configurations.